analytics_fetcher/support/utils.py

```diff
@@ -4,7 +4,8 @@
 
 def base64_encode(bs):
```

Comment:
Could we rename this param please `byte_string` maybe?

Reply:
done ✅  @gclssvglx